keyboards/pabile/p18/keymaps/macro_layer/keymap.c

```diff
@@ -0,0 +1,139 @@
+ -0,0 +1,77 @@
```

Comment:
```suggestion
```

And could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15